decidim-core/app/controllers/decidim/links_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Decidim
+  class LinksController < Decidim::ApplicationController
+    helper Decidim::ExternalDomainHelper
+
+    def new; end
```

Comment:
We should add a header for the search bots on this page asking them not to index the external link warning pages:

```ruby
headers["X-Robots-Tag"] = "noindex"
```